db/migrate/20210707014011_add_patient_state_indexes.rb

```diff
@@ -0,0 +1,10 @@
+class AddPatientStateIndexes < ActiveRecord::Migration[5.2]
+  def change
+    add_index :reporting_patient_states, [:assigned_facility_region_id], name: "patient_states_assigned_facility"
+    add_index :reporting_patient_states, [:assigned_block_region_id], name: "patient_states_assigned_block"
+    add_index :reporting_patient_states, [:assigned_district_region_id], name: "patient_states_assigned_district"
+    add_index :reporting_patient_states, [:assigned_state_region_id], name: "patient_states_assigned_state"
+    add_index :reporting_patient_states, [:patient_id, :month_date], unique: true, name: "patient_states_patient_id_month_date"
```

Comment:
This allows us to `refresh concurrently` for this particular materialized view.  We can probably add unique indexes on the other matviews as well so they can refresh concurrently, but I wanted to keep this PR small and focused.